src/System.Text.Primitives/System/Text/Formatting/PrimitiveFormatter_time.cs

```diff
@@ -182,19 +182,19 @@ static bool TryFormatDateTimeFormatO(DateTimeOffset value, bool isDateTime, Span
         {
```

Comment:
The parameter names are very inconsistent. We need to settle on the name for EncodingData and then rename all these parameters. I just renamed a couple opportunistically.